### PR TITLE
[[ Bug 17622 ]] Encode/decode html text transferred on the clipboard

### PR DESCRIPTION
**Pull request by livecodeian**

### CODE REVIEW

File: engine/src/lnx-clipboard.h [hunk 1]
```diff
@@ -112,8 +112,8 @@ class MCLinuxRawClipboard :
     virtual bool FlushData();
     virtual uindex_t GetMaximumItemCount() const;
     virtual MCStringRef GetKnownTypeString(MCRawClipboardKnownType p_type) const;
-    virtual MCDataRef EncodeFileListForTransfer(MCStringRef p_file_list) const;
-    virtual MCStringRef DecodeTransferredFileList(MCDataRef p_data) const;
+	virtual MCDataRef EncodeHTMLFragmentForTransfer(MCDataRef p_html) const;
+	virtual MCDataRef DecodeTransferredHTML(MCDataRef p_html) const;
```
Comment (livecodefraser) [inline engine/src/lnx-clipboard.h:119]: Probably shouldn't remove the file list methods from here

**Comment by livecodefraser** [comment]
@livecode-vulcan review okay 0a4a567

**Comment by livecode-vulcan** [comment]
💙 review by @livecodefraser ok 0a4a56708f435edc2bd7a9232034992a7a48aa39

**Comment by livecode-vulcan** [comment]
😎 test success 0a4a56708f435edc2bd7a9232034992a7a48aa39
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/236)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/240)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/236)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/236)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/240)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/236)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/236)

**Pull request merged**